cmake/onnxruntime_providers.cmake

```diff
@@ -674,7 +674,7 @@ if (onnxruntime_USE_TENSORRT)
     set(onnxparser_link_libs nvonnxparser_static)
   endif()
 
-  set(trt_link_libs cudnn cublas ${CMAKE_DL_LIBS} ${TENSORRT_LIBRARY})
+  set(trt_link_libs onnxruntime_framework cudnn cublas ${CMAKE_DL_LIBS} ${TENSORRT_LIBRARY})
```

Comment:
this looks a little weird.  we shouldn't need to do this.
we should be going through the provider bridge to access ort framework stuff?
other EP's don't do this?